shell/key-bindings.fish

```diff
@@ -84,4 +86,26 @@ function fzf_key_bindings
     bind -M insert \cr fzf-history-widget
     bind -M insert \ec fzf-cd-widget
   end
+
+  function __fzf_get_starting_dir
+    set -l dir (commandline -t)
+
+    # Default to current dir if no argument input
+    if [ -z "$dir" ]
+        set dir '.'
+    end
+
+    # The commandline token might be escaped, we need to unescape it.
+    set dir (eval "printf '%s' $dir")
```

Comment:
Not required if the above condition is met, and we can put it in the else clause.